### DISCUSSION TITLE
Communities: Webhooks support

### DISCUSSION THREAD

**Discussion by lnielsen** [inveniosoftware/product-rdm#160] [Ideas]

**Comment by Herrner** [comment]
what should these do?

**Reply by lnielsen** [reply]
We'll be sharing some more info in the next telecon, but essentially it could be to run third-party checks on records when submitted to a community. E.g. run a preservation file format check, run FAIR check, extract and process something from the record. 

The other use cases is inform other systems about events happening in the repository - e.g. a new record was accepted into a community, so they can act on it.

**Comment by lnielsen** [comment]
Documented to certain uses cases here: https://doi.org/10.5281/zenodo.10901185
